extensions/amp-megaphone/0.1/test/validator-amp-megaphone.out

```diff
@@ -0,0 +1,101 @@
+|      <script async custom-element="amp-megaphone" src="https://cdn.ampproject.org/v0/amp-megaphone-0.1.js"></script>
+|       <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|      <script async src="https://cdn.ampproject.org/v0.js"></script>
+|  </head>
+|  <body>
+|    <!-- valid: classic mode -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951"
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: tile mode -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-tile="true"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:36:2 The attribute 'data-tile' in tag 'amp-megaphone [data-episode]' is set to the invalid value 'true'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: light mode -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-light="true"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:39:2 The attribute 'data-light' in tag 'amp-megaphone [data-episode]' is set to the invalid value 'true'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: integer start time -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-start="56"
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- valid: decimal start time -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-start="300.32"
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- invalid: bad start time -->
+|    <amp-megaphone height=166 data-episode="OSC7749686951" data-start="300s"
+>>   ^~~~~~~~~
+amp-megaphone/0.1/test/validator-amp-megaphone.html:48:2 The attribute 'data-start' in tag 'amp-megaphone [data-episode]' is set to the invalid value '300s'. (see https://amp.dev/documentation/components/amp-megaphone) [DISALLOWED_HTML]
+|                    layout="fixed-height"></amp-megaphone>
+|    <!-- invalid: light mode must be 'true' or 'false'-->
+|    <amp-megaphone height=166 data-trackid="243169232" data-light
```

Comment:
Is there intended to be a default, such as data-episode, if neither of the type attrs are present? If not, maybe fix this test case to include one of those so the error matches the intended issue.

If so, then I'd remove mandatory=true and dispatch_key on the default version tagspec only.